src/app.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Form from './form'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      question: '',
+      answer: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSave = this.handleSave.bind(this)
+  }
+  handleChange(event) {
+    const form = event.target
+    this.setState({
+      [form.name]: form.value
+    })
+  }
+  handleSave(event) {
+    event.preventDefault()
+    console.log(this.state)
```

Comment:
You should put these in your `App` state as the user adds them. Use the React DevTools to confirm that they're added properly. Then be sure to `reset` the form.